pageserver/src/page_service.rs

```diff
@@ -200,6 +202,12 @@ async fn page_service_conn_main(
         gauge.dec();
     }
 
+    fail::fail_point!(
+        "simulated-bad-compute-connection",
+        rand::random::<f64>() < CONNECTION_FAILURE_PROBABILITY,
```

Comment:
I think we can avoid hardcoding `CONNECTION_FAILURE_PROBABILITY` & get it in the failpoint configuration that's passed in when enabling it:

```
The format of a single action is [p%][cnt*]task[(arg)]. p% is the expected probability that the action is triggered, and cnt* is the max times the action can be triggered. The supported values of task are:
```

https://docs.rs/failpoints/latest/failpoints/fn.cfg.html

Reply:
Nice, I missed that somehow. Changed it